File: dev-docs/snippets/documents/transfer-ownership.swift
import JsBaoClient

// Transfer document ownership to another user. Swift returns an untyped
// `[String: Any]` (vs JS `void`).
func transferOwnership(
  client: JsBaoClient, documentId: String, newOwnerId: String
) async throws {
  // #region example
  let result = try await client.documents.transferOwnership(
    documentId: documentId, newOwnerId: newOwnerId
  )
  // #endregion example
  _ = result
}

File: dev-docs/snippets/documents/transfer-ownership.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Transfer document ownership to another user.
export async function transferOwnership(
  client: JsBaoClient,
  documentId: string,
  newOwnerId: string,
) {
  // #region example
  await client.documents.transferOwnership(documentId, newOwnerId);
  // #endregion example
}
